edgelet/edgelet-core/src/settings.rs

```diff
@@ -354,5 +375,57 @@ pub trait RuntimeSettings {
     fn homedir(&self) -> &Path;
     fn certificates(&self) -> Option<&Certificates>;
     fn watchdog(&self) -> &WatchdogSettings;
-    fn diff_with_cached(&self, path: &Path) -> bool;
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Settings<T: Clone> {
```

Comment:
Remove the `T: Clone` bound here.

Reply:
I think I put that bound because I thought the `#[derive(Clone)` will complain otherwise. I see that it actually doesn't. Will remove.

Reply:
Yes, all the compiler built-in `#[derive]`s emit the impls with the same bounds on the type parameters.

It's actually a problem when you *don't* want that, such as for a struct that contains `Arc<T>` where the struct's Clone does *not* need T to be Clone.